### PR TITLE
Remove button

### PR DESCRIPTION
**Pull request by Vanethos**
Changed the "Use previous title/description" button so that the button is not shown on the first time the user is uploading. #503 #716

### CODE REVIEW

File: app/src/main/java/fr/free/nrw/commons/upload/SingleUploadFragment.java [hunk 1]
```diff
@@ -101,6 +101,11 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         prefs = PreferenceManager.getDefaultSharedPreferences(getActivity());
         license = prefs.getString(Prefs.DEFAULT_LICENSE, Prefs.Licenses.CC_BY_SA_3);
 
+        // check if this is the first time we have uploaded
+        if (prefs.getString("Title", "").trim().length() == 0 && prefs.getString("Desc", "").trim().length() == 0) {
```
Comment (whym) [inline app/src/main/java/fr/free/nrw/commons/upload/SingleUploadFragment.java]: The style checker says this line is too long. Can you perhaps split it at the && operator?

Comment (whym) [inline app/src/main/java/fr/free/nrw/commons/upload/SingleUploadFragment.java]: @Vanethos What do you think? I admit this is a minor issue. Maybe we can leave it for future (mass) fix.

**Comment by misaochan** [comment]
Yeah, I'm good with merging this if you are, @whym .

File: app/src/main/java/fr/free/nrw/commons/upload/SingleUploadFragment.java (hunk 1 above)
Comment (Vanethos) [inline app/src/main/java/fr/free/nrw/commons/upload/SingleUploadFragment.java]: @whym  I'm deeply sorry for not answering before! I will split the comment and try to use the same pull request (still have to check how to do it)

**Comment by Vanethos** [comment]
I have edited the file directly in Github, hope that this solves the issue!

File: app/src/main/java/fr/free/nrw/commons/upload/SingleUploadFragment.java [hunk 2]
```diff
@@ -188,8 +194,8 @@ public void onDestroyView() {
         String desc = titleDesc.getString("Desc", "");
         Timber.d("Title: %s, Desc: %s", title, desc);
 
-        titleEdit.setText(title);
         descEdit.setText(desc);
+        titleEdit.setText(title);
```
Comment (misaochan) [inline app/src/main/java/fr/free/nrw/commons/upload/SingleUploadFragment.java]: Is there a reason why this was flipped?

File: app/src/main/java/fr/free/nrw/commons/upload/SingleUploadFragment.java [hunk 3]
```diff
@@ -101,6 +101,12 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         prefs = PreferenceManager.getDefaultSharedPreferences(getActivity());
         license = prefs.getString(Prefs.DEFAULT_LICENSE, Prefs.Licenses.CC_BY_SA_3);
 
+        // check if this is the first time we have uploaded
+        if (prefs.getString("Title", "").trim().length() == 0 &&
```
Comment (misaochan) [inline app/src/main/java/fr/free/nrw/commons/upload/SingleUploadFragment.java]: Minor issue, but Codacy wants `&&` to be on the next line

File: app/src/main/java/fr/free/nrw/commons/upload/SingleUploadFragment.java (hunk 2 above)
Comment (whym) [inline app/src/main/java/fr/free/nrw/commons/upload/SingleUploadFragment.java]: Yeah, this looks inconsistent. Title is above description both in the variable declaration and the UI.

**Comment by whym** [comment]
I went ahead and applied the last two suggestions. If it's okay, I'd like to squash these into one commit and merge it.

**Comment by misaochan** [comment]
@whym great, go for it!

**Comment by misaochan** [comment]
(oops, forgot that I could squash and merge on my phone, haha)

**Pull request merged**